Makefile

```diff
@@ -41,11 +45,19 @@ ENTRY_POINT := _rt0_arm_tamago
 ARCH = "arm"
 
 GOFLAGS = -tags ${BUILD_TAGS} -trimpath \
-        -ldflags "-T ${TEXT_START} -E ${ENTRY_POINT} -R 0x1000 -s -w \
-                  -X 'main.Build=${BUILD}' -X 'main.Revision=${REV}' -X 'main.Version=${BUILD_EPOCH}' \
+        -ldflags "-T ${TEXT_START} -E ${ENTRY_POINT} -R 0x1000 \
+                  -X 'main.Build=${BUILD}' -X 'main.Revision=${REV}' -X 'main.Version=${GIT_SEMVER_TAG}' \
                   -X 'main.PublicKey=$(shell test ${PUBLIC_KEY} && cat ${PUBLIC_KEY} | tail -n 1)' \
                   -X 'main.GitHubUser=${GITHUB_USER}' -X 'main.GitHubEmail=${GITHUB_EMAIL}' -X 'main.GitHubToken=${GITHUB_TOKEN}' \
-                  -X 'main.RestDistributorBaseURL=${REST_DISTRIBUTOR_BASE_URL}'"
+                  -X 'main.RestDistributorBaseURL=${REST_DISTRIBUTOR_BASE_URL}' \
+                  -X 'main.updateBinariesURL=${FT_BIN_URL}' \
+                  -X 'main.updateLogURL=${FT_LOG_URL}' \
+                  -X 'main.updateLogOrigin=${FT_LOG_ORIGIN}' \
+                  -X 'main.updateLogVerifier=$(shell cat ${LOG_PUBLIC_KEY})' \
+                  -X 'main.updateAppletVerifier=$(shell cat ${APPLET_PUBLIC_KEY})' \
+                  -X 'main.updateOSVerifier1=$(shell cat ${OS_PUBLIC_KEY1})' \
+                  -X 'main.updateOSVerifier2=$(shell cat ${OS_PUBLIC_KEY2})' \
+				  "
```

Comment:
nit: what's that double quote doing all the way out over there?

Reply:
It was probably on a voyage of self-discovery, pondering the eternal dilemma of tabs vs. spaces...